lib/web3.js

```diff
@@ -2,7 +2,7 @@ import Web3 from 'web3';
 import contract from 'truffle-contract';
 import _ from 'lodash';
 
-const provider = new Web3.providers.HttpProvider('http://localhost:8545');
+const provider = new Web3.providers.HttpProvider('http://165.227.28.206:8545');
```

Comment:
I ended up running a local one to simplify things the other day.  Ultimately we will have to attach to a node.